test/IRGen/enum_value_semantics.sil

```diff
@@ -76,7 +76,7 @@ enum MultiPayloadGeneric<T> {
 enum MultiPayloadNontrivialSpareBits {
   case payload1(Builtin.NativeObject)
   case payload2(Builtin.Int64)
-  case payload3(Builtin.Int64, Builtin.UnknownObject)
+  case payload3(Builtin.Int64, Builtin.NativeObject)
```

Comment:
These tests relied on having spare bits in UnknownObject, which was a questionable assumption. (Good thing we weren't actually using it!)